### DISCUSSION TITLE
How login to account without code?

### DISCUSSION THREAD

**Discussion by vidone-omid-git** [community/community#101358] [New to GitHub]
I used Google Authenticator to verify GitHub login but most recently, my device was lost, and I cannot access it.
Also, I don't know my recovery code.
How can I solve the problem?

**Comment by JCCourt** [comment]
Contact support to get the 2FA removed

https://support.github.com/features/account

**Comment by vidone-omid-git** [comment]
> Contact support to get the 2FA removed
> 
> https://support.github.com/features/account

I tried it but I cannot send a message because GitHub show Virtual Assistant page

**Reply by JCCourt** [reply]
You can find more information here..

https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials

But I believe it's the only way.

**Comment by vidone-omid-git** [comment]
> You can find more information here..
> 
> https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials
> 
> But I believe it's the only way.

I followed all way for recovery but I cannot do it
I have ssh keys, Can I recover with it?

**Comment by ghostinhershell** [comment]
Hi @vidone-omid-git, thanks so much for reaching out in our community. 

We do have a support article that documents next steps if you lose access to your two-factor authentication credentials: [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). I think this section, specifically, should be helpful: [Requesting help with two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#requesting-help-with-two-factor-authentication).

If that doesn't help, you can submit a support ticket via our [Support page](https://support.github.com/contact).
